classes/controller/FrontController.php

```diff
@@ -1674,7 +1695,7 @@ public function getTemplateVarPage()
             'lang-rtl' => (bool) $this->context->language->is_rtl,
             'country-' . $this->context->country->iso_code => true,
             'currency-' . $this->context->currency->iso_code => true,
-            $this->context->shop->theme->getLayoutNameForPage($this->php_self) => true,
+            $this->getLayoutName() => true,
```

Comment:
If I understand right, when using `getLayoutNameForPage` it was always providing the default layout, without checking $content_only param, right?

Reply:
Yes, exactly.

AND

It was using $this->php_self when calling it, but this parameter is not used for module front controllers.

Reply:
>It was using $this->php_self when calling it, but this parameter is not used for module front controllers.

https://github.com/prestarocket/morinella/blob/6abd895a45afc0d9383d4012f2a768bc429793e4/modules/leoblog/controllers/front/category.php#L35

Reply:
Hmmm, OK, maybe it could be used. But if not, there was no fallback. :-)